spectec/spec/wasm-1.0/8-reduction.watsup

```diff
@@ -15,22 +15,24 @@ rule Step/read:
   -- Step_read: z; instr* ~> instr'*
 
 
-;; Expressions
-
-relation Eval: config ~>* state; val*  hint(show "E-expr")
-relation Eval_expr: state; expr ~>* state; val*  hint(show "E-expr")
+relation Steps: config ~>* config  hint(show "E")
```

Comment:
Nit: move up to other relations.

Reply:
In that case, should I also move rules for `Steps` above rules for `Step`?

Reply:
I would move the relation decl after the others, so the rules can stay here, I think.

Reply:
Oh I see, I thought you meant moving `Steps` relation above other `Step` relations.